server/initializers/database.ts

```diff
@@ -40,6 +41,11 @@ const sequelizeTypescript = new SequelizeTypescript({
   port,
   username,
   password,
+  pool: {
+    max: poolMax,
+    idle: 30000,
+    acquire: 60000
```

Comment:
Where those values come from?

Reply:
You're right, I should use the default pooling config: https://github.com/sequelize/sequelize/blob/63d4aaa3298e7a8a9b12d96deef1b8d015af712a/lib/dialects/abstract/connection-manager.js#L10-L17